src/map/skill.c

```diff
@@ -12257,7 +12337,10 @@ int skill_unit_onplace_timer(struct skill_unit *src, struct block_list *bl, int6
 			// Aegis style
 			// Check if song has enough time to survive the next check
 			if (!(battle_config.song_timer_reset) && tsc && tsc->data[type] && tsc->data[type]->val4 == 1) {
-				const struct TimerData *td = timer->get(tsc->data[type]->timer);
+				const struct TimerData *td;
+				if (tsc == NULL)
```

Comment:
`tsc` is already nonnull (see `if (!(battle_config.song_timer_reset) && tsc && tsc->data[type] && tsc->data[type]->val4 == 1) {` above)